developers/weaviate/config-refs/schema/index.md

```diff
@@ -150,6 +150,26 @@ To avoid this, you can either:
 
 We are working on a re-indexing API to allow you to re-index the data after adding a property. This will be available in a future release.
 
+### Collections count limit {#collections-count-limit}
+
+:::info
+Added in `v1.25.34`, `v1.26.18`, `1.27.15`, `1.28.9`, `1.29.2`
+:::
+
+To ensure optimal performance, Weaviate **limits the number of collections per instance**. Each collection adds overhead in terms of indexing, definition management, and storage. By setting a default limit, we help maintain performance and reduce operational complexity.
```

Comment:
> By setting a default limit, we help maintain performance and reduce operational complexity.

Wdyt about "This limit aims to ensure Weaviate remains performant."

Then I propose to move the admonition so it is the following paragraph. Because the content of the admonition may be what the reader is looking for (i.e. Oh, but I want N collections for my use case).

Reply:
Agreed, I simplified the wording. 
Which admonition are you referring to, the note (if you already exceed the limit) or the tip (rethink your architecture)?